application/comit_node/src/http_api/routes/rfc003/handlers/get_action.rs

```diff
@@ -166,30 +166,32 @@ impl IntoResponseBody for bitcoin::SpendOutput {
                             network,
                         }),
                         Err(e) => {
-                            error!("Could not serialized signed Bitcoin transaction: {:?}", e);
+                            error!("Could not serialize signed Bitcoin transaction: {:?}", e);
                             Err(HttpApiProblem::with_title_and_type_from_status(
                                 StatusCode::INTERNAL_SERVER_ERROR,
                             )
-                            .set_detail("Issue encountered when serializing Bitcoin transaction"))
+                            .set_detail("Issue encountered when serializing Bitcoin transaction."))
                         }
                     }
                 }
                 Err(_) => Err(HttpApiProblem::with_title_and_type_from_status(
                     StatusCode::BAD_REQUEST,
                 )
-                .set_detail("fee-per-byte is not a valid float")),
+                .set_title("Invalid query parameter.")
+                .set_detail("Query parameter fee-per-byte is not a valid float.")),
             },
             _ => {
-                error!("Unexpected GET parameters for a bitcoin::SpendOutput action type. Expected: address and fee-per-byte.");
+                error!("Unexpected GET parameters for a bitcoin::SpendOutput action type. Expected: address and fee-per-byte");
                 let mut problem =
                     HttpApiProblem::with_title_and_type_from_status(StatusCode::BAD_REQUEST)
-                        .set_detail("This action requires additional query parameters");
+                        .set_title("Missing query parameter(s).")
```

Comment:
:wrench: This one could also be extracted into its own function, similar to `unexpected_query_parameters`.

However, instead of flattening out the parameters, it is would probably be good to have them under a single property like `missing_query_parameters`.